packages/fetch-cached-dns/.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  extends: [require.resolve("@vercel/style-guide/eslint")],
+  rules: {
+    "no-bitwise": "off",
+    "no-param-reassign": "off",
```

Comment:
As this is part of our style guide, it would be better to fix these - or leave TODOs to fix the issues.

Turning rules on/off at a package level is going to be hard to maintain long-term.

Reply:
https://github.com/vercel/fetch/pull/67#discussion_r836746430